src/commcare_cloud/ansible/roles/lpar2rrd/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-- include: install_agent.yml
+- include_tasks: install_agent.yml
   tags: lpar2rrd_agent
-- include: install_master.yml
+- include_tasks: install_master.yml
   when: inventory_hostname in groups.lpar2rrd_master|default([])
```

Comment:
The first task (install_agent.yml) is a static task because it is included unconditionally and does not depend on any dynamic conditions or variables.

The second task (install_master.yml) is a dynamic task because it is included conditionally based on the value of the inventory_hostname variable. If the inventory_hostname is in the group specified by groups.lpar2rrd_master, then the task is included; otherwise, it is skipped.

Therefore, while the first task is static, the second task is dynamic.